app/views/layouts/admin_layout.html.erb

```diff
@@ -10,27 +10,27 @@
 
   <%= render "govuk_publishing_components/components/skip_link" %>
   <%= render "govuk_publishing_components/components/layout_header", {
-    product_name: "Publisher",
+    product_name: "",
```

Comment:
Can you fix this up? And then ignore my questions about the product name in the first commit.